api/src/Modules/staff/App/Dependencies.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+use Slim\App;
+use App\Modules\staff\Service\StaffService;
+use App\Modules\staff\Repository\StaffRepository;
+
+/* setupStaffAPIDependencies */
+
+function setupStaffAPIDependencies(App $app, array $settings)
+{
+    $container = $app->getContainer();
+
+    $container['StaffService'] = function ($c): StaffService {
+        return new StaffService(
+            $c->get('EventService'),
+            $c->get('DepartmentService'),
+            $c->get('MemberService'),
+            new StaffRepository($c['db'])
+        );
+    };
+
+}
```

Comment:
Same here,  Have these 2 changes go in with the changes to the `Staff` module.

Reply:
Sure, no problem, I can make the adjustment there.